pcli/src/command/view/address.rs

```diff
@@ -28,13 +31,21 @@ impl AddressCmd {
             true => fvk.incoming().ephemeral_address(OsRng, self.index.into()),
         };
 
-        if self.base64 {
-            println!(
-                "{}",
-                base64::engine::general_purpose::STANDARD.encode(address.to_vec())
-            );
-        } else {
-            println!("{}", address);
+        let address_index = fvk.incoming().address_index(&address);
+
+        match address_index {
+            Some(address_index) => {
+                if self.base64 {
+                    println!(
+                        "Account {0}: {1}",
+                        address_index.account,
+                        base64::engine::general_purpose::STANDARD.encode(address.to_vec())
+                    );
```

Comment:
I think this breaks the base64 output format by making it text-based instead.

Reply:
It doesn't seem to based on testing this implementation alongside the legacy implementation - both base64 address strings match.